api.planx.uk/gis/classifiedRoads.ts

```diff
@@ -34,6 +34,7 @@ type PlanningConstraintBody = {
   value: boolean;
   text: string;
   data?: OSFeatures["features"];
+  category?: string;
```

Comment:
Are categories actually an enum? 

Since the plan sounds like it would be to use these strings as semantic tags in the constraints component, then I think a string enum or constant object would be the way to go since it describes the range of possibilities and allows for exhaustive checking. Types can be great as documentation for things like this.

Reply:
I _think_ so, but actually need to confirm this with content team! Similar to above, will address when the main planning constraints API file migrates to typescript (it'd be a bit of a false enum if only referenced in the "roads" file here because the roads response should always be the _same_ category).